k8s/prod/common/em/em-anno.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: em-anno
+  namespace: em
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.em-anno: glob:prod-*
+spec:
+  releaseName: em-anno
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: em-anno
+    version: 0.1.11
+  values:
+    anno:
+      replicas: 2
+      applicationInsightsInstrumentKey: "039faf8c-4682-432e-8fc5-f9735bb4b962"
```

Comment:
You can remove this , if you make a change to bootstrap.yaml like this https://github.com/hmcts/draft-store/blob/master/src/main/resources/bootstrap.yaml#L8